package.json

```diff
@@ -73,10 +70,12 @@
     "cytoscape-popper": "1.0.2",
     "d3-format": "1.3.2",
     "deep-freeze": "0.0.1",
+    "jquery": "3.5.0",
     "js-yaml": "3.13.1",
     "json-beautify": "1.0.1",
-    "lodash": "4.17.11",
+    "lodash": "4.17.15",
```

Comment:
Using `yarn why lodash | grep Found` still lists version 4.17.11. I'm not sure why. Perhaps you manually edited this line and there after did `yarn install`? Perhaps you could try `yarn upgrade lodash@4.17.15` to see if the deps also get the library update. Otherwise, probably you need to duplicate this line to the `resolutions` part.

Reply:
OK, thanks.